index.ejs

```diff
@@ -48,7 +48,9 @@
     <link href="<%= htmlWebpackPlugin.files.css[key] %>" rel="stylesheet">
     <% } %>
     <% } %>
-
+    <% if (htmlWebpackPlugin.options.headHtmlSnippet) { %>
+      <%= htmlWebpackPlugin.options.headHtmlSnippet %>
+    <% } %>
```

Comment:
It's tricky to find the right placement for this - e.g., if we want to inject `<meta>` tags, it might be nicer higher up in `<head>`, but if we want to inject styles that load after the ones Webpack inserts, then it makes sense below here.